metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_classes.py

```diff
@@ -182,6 +184,7 @@ def new_powerbi_dataset(workspace_id: str, raw_instance: dict) -> PowerBIDataset
     return PowerBIDataset(
         id=raw_instance["id"],
         name=raw_instance.get("name"),
+        description=raw_instance.get("description"),
```

Comment:
Can we change it to `description=raw_instance.get("description", str())` so that we don't need to handle it in higher level python module

Reply:
Fixed!